products/mysql.md

```diff
@@ -2,7 +2,7 @@
 title: MySQL
 category: db
 sortReleasesBy: "releaseCycle"
-changelogTemplate: "https://dev.mysql.com/doc/relnotes/mysql/__RELEASE_CYCLE__/en/news-__LATEST_SHORT_HAND__.html"
+changelogTemplate: "https://dev.mysql.com/doc/relnotes/mysql/__RELEASE_CYCLE__/en/news-{{"__LATEST__" | replace:'.','-'}}.html"
```

Comment:
Ok, the YAML parser isn't a fan of using double quotes inside double quotes, I get it...